api/src/main/java/marquez/service/models/Run.java

```diff
@@ -16,29 +16,41 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 import com.google.common.collect.ImmutableMap;
 import java.time.Instant;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 import javax.annotation.Nullable;
+import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NonNull;
 import lombok.Setter;
 import lombok.ToString;
 import lombok.extern.slf4j.Slf4j;
+import marquez.api.models.JobVersion;
 import marquez.common.models.DatasetVersionId;
 import marquez.common.models.JobName;
 import marquez.common.models.JobVersionId;
 import marquez.common.models.NamespaceName;
 import marquez.common.models.RunId;
 import marquez.common.models.RunState;
 
+/**
+ * Service model for a job run. Job attributes are serialized as {@link JobVersion} attributes. The
+ * {@link #location} field is ignored entirely in the serialized output. Deserializing from JSON
```

Comment:
Why is the location missing when serialized?

Reply:
Actually, I don't know. Maybe you remember? 😉 https://github.com/MarquezProject/marquez/commit/524abc80879c1967ecaaf41951d1d58cebd576df